### PR TITLE
Distinguish between archive and access bags

### PR DESCRIPTION
**Pull request by jtweed**
### What is this PR trying to achieve?

Make the distinction between archive and access assets clearer, by storing them as different bags. This allows us to use spaces to decide which locations a specific bag should stored.

### Who is this change for?

@mklander's sanity

### CODE REVIEW

**Comment by jtweed** [comment]
What do we reckon, should I create a ticket to do this?

File: docs/rfcs/002-archival_storage/README.md [hunk 1]
```diff
@@ -40,15 +39,17 @@ We'll need to integrate with other services such as:
...
 
-These services will need to provide accessions in the BagIt format, zip-compressed and uploaded to an S3 bucket. They should then call an ingest API and provide a callback URL that will be notified when the ingest has succeeded or failed.
+These services will need to provide assets in the BagIt format, zip-compressed and uploaded to an S3 bucket. They should then call an ingest API and provide a callback URL that will be notified when the ingest has succeeded or failed.
+
+When there is a distinction between archival and access assets, these should be submitted as separate bags. This allows storing archival assets and access assets in different kinds of storage.
 
 ### Storage
 
-All assets will be stored in S3 Infrequent Access and replicated to S3 Glacier. We will store a second replica in Azure Blob Storage with the Archive storage class.
+Archival bags will be stored in S3 with a Glacier storage class and replicated to Azure Blob Storage with the Archive storage class. Access bags will be stored in S3 with an Infrequent Access storage class.
```
Comment (alexwlchan) [inline docs/rfcs/002-archival_storage/README.md]: Do we plan to put archive bags in multiple buckets in S3?

File: docs/rfcs/002-archival_storage/README.md [hunk 2]
```diff
@@ -40,15 +39,17 @@ We'll need to integrate with other services such as:
...
 
-S3 Glacier storage will have versioning enabled, but we will only keep the most recent version in S3 Infrequent Access.
+AWS storage will have versioning enabled, but we will only keep the most recent version in Azure as it is intended only for worst case disaster recovery following a complete failure of AWS.
 
-The S3 Glacier version should be stored for every file. The S3 Glacier version of the tagmanifest should be used as the overall version of the bag.
+The underlying storage provider version should be stored for every file. The underlying storage provider version of the tagmanifest should be used as the overall version of the bag.
```
Comment (alexwlchan) [inline docs/rfcs/002-archival_storage/README.md]: I’m not sure I understand this paragraph – stored how and where?

**Comment by alexwlchan** [comment]
> What do we reckon, should I create a ticket to make this change?

I think so. And do we need a ticket for the last round of changes to the /bags API? I think we're still presenting separate access/archive locations.

**Comment by kenoir** [comment]
LGTM barring @alexwlchan comments.

**Pull request merged**